web-ng/src/app/components/map/map.component.ts

```diff
@@ -211,22 +201,9 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     }
   }
 
-  private addFeaturesToMap(project: Project, features: List<Feature>) {
-    const locationFeatureIds = Array.from(this.markers.keys());
-    const geoJsonFeatureIds: String[] = [];
-    const polygonFeatureIds = Array.from(this.polygons.keys());
-    this.map.data.forEach(f => {
-      geoJsonFeatureIds.push(f.getProperty('featureId'));
-    });
-    this.map.data.addListener('click', (event: google.maps.Data.MouseEvent) => {
-      const featureId = event.feature.getProperty('featureId');
-      if (this.disableMapClicks) {
-        return;
-      }
-      this.zone.run(() => {
-        this.navigationService.selectFeature(featureId);
-      });
-    });
+  private addNewFeatures(project: Project, features: List<Feature>) {
```

Comment:
Since some of these methods do non-trivial work, it may be worth adding minimal JSDoc descriptions (can leave off args description if they're clear).